knack/iframeMapMessenger.js

```diff
@@ -79,9 +72,12 @@
       if (data.message === "LAT_LON_FIELDS") {
         console.log("received message ", data);
         var $latLonFields = $("#kn-input-field_3300");
-
-        $latLonFields.find("#latitude").val(data.lat);
-        $latLonFields.find("[name='longitude']").val(data.lng);
+        if (!!data.lat && !!data.lng) {
+          $latLonFields.find("#latitude").val(data.lat);
+          $latLonFields.find("[name='longitude']").val(data.lng);
+        } else {
+          console.error("Payload missing complete location data ", data);
```

Comment:
when/why would does this happen?

Reply:
I am going to remove this because I dont think it will ever happen. When you were clicking the "Add location" in previous PRs, I thought it was a failure in the iframe message communication, but the failure was the NextJS app wasnt even attemping to send knack a message and knack was saving empty fields as a location.